### ISSUE TITLE
Will you post the training code, and trained model for ImageNet?

### ISSUE THREAD

**Issue by songyh10** [satoshiiizuka/siggraph2017_inpainting#1]
Hi, thanks for your providing the test code. Do you have plan to post the training code and the trained models for more datasets other than the Place2 dataset? Many thanks!

**Comment by unlugi** [comment]
Yes, I have the same problem. Where is the training code? Are the authors planning on making it open source?

**Comment by GuardSkill** [comment]
Yes, I have the same problem. Are the authors planning on making it open source?

**Comment by iakhiljoy** [comment]
facing the same problem...

**Comment by Zxl19990529** [comment]
> facing the same problem...

Hi, can you download the pre-trained model `completionnet_places2.t7` ? I can't download it via the script  `download_model.sh` because of the network. Could you please send me the model? Many thanks.

This is my e-mail:  `zhangxinliang_mit@163.com`

**Comment by Zxl19990529** [comment]
> Yes, I have the same problem. Where is the training code? Are the authors planning on making it open source?

Hi, can you download the pre-trained model `completionnet_places2.t7` ? I can't download it via the script `download_model.sh` because of the network. Could you please send me the model? Many thanks.

This is my e-mail: `zhangxinliang_mit@163.com`

**Comment by lelandyang** [comment]
The author uploaded training code in an archive to his website about 2 years ago (2017) and he certainly regretted for open sourcing it and thus only the prediction model was available here. SHAME ON HIM.
